src/main/java/net/sf/rails/algorithms/RevenueAdapter.java

```diff
@@ -375,14 +375,13 @@ private int maxTravelEdges() {
         for (EdgeTravel edgeTravel:edgeTravelSets.values()) {
             maxNbEdges = Math.max(maxNbEdges, edgeTravel.set.size());
         }
-        for ( NetworkEdge edge : edgeTravelSets.keySet() ) {
-            EdgeTravel edgeTravel = edgeTravelSets.get(edge);
+        for ( Map.Entry<NetworkEdge, EdgeTravel> entry : edgeTravelSets.entrySet() ) {
             StringBuilder edgeString = new StringBuilder("RA: EdgeSet for ").
-                    append(edge.toFullInfoString()).
+                    append(entry.getKey().toFullInfoString()).
                     append(" size = ").
-                    append(edgeTravel.set.size()).
+                    append(entry.getValue().set.size()).
```

Comment:
I agree using `Set#entrySet` is better here. Still I think we should give both the key and the value a better name before working with them by adding something similar to:

```java
final NetworkEdge edge = entry.getKey();
final EdgeTravel edgeTravel = entry.getValue();
```

Reply:
Man, I had to introduce a variable if it's only used once (as in here). I do see the value if it's used multiple times... do you really think we should still change this?